backend/fms_core/models/sample.py

```diff
@@ -62,6 +64,10 @@ def is_pool(self) -> bool:
     def derived_sample_not_pool(self) -> DerivedSample:
         return self.derived_samples.first() if not self.is_pool else []  # Forces crash if pool
 
+    @property
+    def biosample(self) -> Biosample:
```

Comment:
Might want to rename it similar to derived_sample_not_pool in order to make sure we remember not to use this on pools.